src/components/itemContextMenu.js

```diff
@@ -516,6 +534,22 @@ function executeCommand(item, id, options) {
             case 'queueallfromhere':
                 getResolveFunction(resolve, id)();
                 break;
+            case 'removefromcontinuewatching':
+                apiClient.ajax({
+                    url: apiClient.getUrl('Users/' + options.user.Id + '/ExcludeContinueWatching/' + item.Id, {}),
+                    type: 'POST'
+                }).then(function () {
+                    getResolveFunction(resolve, id, true)();
+                });
+                break;
+            case 'removefromnextup':
+                apiClient.ajax({
+                    url: apiClient.getUrl('Users/' + options.user.Id + '/ExcludeNextUp/' + item.Id, {}),
+                    type: 'POST'
+                }).then(function () {
+                    getResolveFunction(resolve, id, true)();
+                });
```

Comment:
IMO, API calls should be moved to utils (say `src/utils/jellyfin-apiclient/nextUp.ts`)

Reply:
That was not the standard followed elsewhere in the code -- API calls are declared in their own functions in the same file, or called inline like we did, there doesn't seem to be a consistent standard of putting calls in utils that's been followed for past PR's. In the style guide it says to respect the surrounding code, and as the surrounding code doesn't define the API functions in the utils dir, nor does the style guide state to, I disagree. 

I don't disagree with this idea, I think it's a good idea, but I think that needs to be a part of a greater refactor where  all the API logic is moved into ..../src/utils/jellyfin-api, or wherever the code should live. 
Example: 
![image](https://github.com/jellyfin/jellyfin-web/assets/16343794/979827c9-2789-43bc-b198-148bbb9dfaef)